admin_ui/backend/tests/test_agents_api.py

```
@@ -342,3 +342,64 @@ def test_reconcile_adds_new_yaml_context(client, tmp_path, monkeypatch):
+    """Load admin_ui/backend/main.py by path so it can't be shadowed by the
+    repo-root main.py (the CLI entrypoint, which has no `app`). Skips when the
+    full backend deps (docker, etc.) aren't installed in this environment."""
+    import importlib.util
+    from pathlib import Path
+
+    main_path = Path(__file__).resolve().parents[1] / "main.py"
+    spec = importlib.util.spec_from_file_location("admin_ui_backend_main", main_path)
+    module = importlib.util.module_from_spec(spec)
+    try:
+        spec.loader.exec_module(module)
+    except Exception as exc:  # missing optional backend deps in this environment
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)